packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/ExtensibleEnumProvider.cs

```diff
@@ -14,12 +14,12 @@
 
 namespace Microsoft.Generator.CSharp
 {
-    public class ExtensibleEnumTypeProvider : EnumTypeProvider
+    public class ExtensibleEnumProvider : EnumProvider
```

Comment:
oh I made a mistake in my PR and it is not found by others - this should be `internal sealed` as well as `FixedEnumProvider`.
Could you please change it here to make it right?